engine/config/config.go

```diff
@@ -324,14 +324,15 @@ func addNodes(c *Cluster, p *pb.Cluster) {
 		nodes = append(nodes, n)
 	}
 	sort.Sort(seesaw.NodesByIPv4{nodes})
+nodesLoop:
 	for i, n := range nodes {
 		switch i {
 		case 0:
 			n.Priority = 255
 		case 1:
```

Comment:
Wouldn't it make more sense to set some priority on each, in case there are more than 2 nodes?

Reply:
as far as I know, Seesaw doesn't support more than 2 nodes.  For example, peer_ip in the config is a single IP. Have you tried to have more than two nodes?

Reply:
I don't see any reason this particular loop couldn't handle more than 2 nodes, in case it is ever supported. What about priorities starting with 255, then remaining nodes n-1 down to 1?

Reply:
I mean other part of the code doesn't support more than 2 nodes. It doesn't make much sense to support multiple nodes just in this loop.  Also you code is not really changing the behavior but just break out of the loop sooner which doesn't seem a must fix. Also this label jump may looks more confusing to people.